app/src/main/res/values-ru/strings.xml

```diff
@@ -344,13 +344,15 @@
   <string name="extension_kbd_top_small_numbers">Цифры</string>
   <string name="extension_kbd_top_numbers_alt">Альтернативные цифровые кнопки</string>
   <string name="extension_kbd_top_nav">Клавиши навигации</string>
+  <string name="extension_kbd_top_numbers_simple">Без лишнего, маленькие цифры</string>
```

Comment:
No need to localize here, use https://crowdin.com/project/anysoftkeyboard for that.